XenModel/Messages.zh-CN.resx

```diff
@@ -3623,19 +3623,19 @@
     <value>yyyy-MM-dd</value>
   </data>
   <data name="DATEFORMAT_DMY_HM" xml:space="preserve">
-    <value>yyyy-MM-dd tt h:mm</value>
+    <value>yyyy-MM-dd HH:mm</value>
   </data>
   <data name="DATEFORMAT_DMY_HMS" xml:space="preserve">
-    <value>yyyy-MM-dd tt h:mm:ss</value>
+    <value>yyyy-MM-dd  HH:mm:ss</value>
   </data>
   <data name="DATEFORMAT_DMY_LONG" xml:space="preserve">
     <value>yyyy年M月d日</value>
   </data>
   <data name="DATEFORMAT_HM" xml:space="preserve">
-    <value>HH:mm</value>
+    <value>tt HH:mm</value>

```

Comment:
I think this should be "tt h:mm", because if you add the "tt" AM/PM designator, then the hour should be represented in the 12-hour clock format ("h" or "hh"), and not in the 24-hour clock format ("HH").

This also applies to line 3638.